views/index.html

```diff
@@ -17,11 +17,99 @@
     <script src="/scripts/app.js"></script>
   </head>
   <body>
-    <div class="container">
+    <div class="jumbotron">
+      <div class="container">
+        <h1>Welcome to Cliffly</h1>
+
+      </div>
+    </div>
+    <section id='album-form' class="container">
+      <div class="row">
+      <div class="col-md-10 col-md-offset-1">
+
+        <form class="form-horizontal" id="singlebutton">
+          <fieldset>
+
+
+            <!-- Form Name -->
+            <legend>Add New Cliff</legend>
+
+            <!-- Text input-->
+            <div class="form-group">
+              <label class="col-md-4 control-label" for="cliffName">Cliff Name</label>
+              <div class="col-md-4">
+              <input id="cliffName" name="cliffName" type="text" placeholder="" class="form-control input-md" required="">
+
+              </div>
+            </div>
+
+            <!-- Text input-->
+            <div class="form-group">
+              <label class="col-md-4 control-label" for="nearestCity">Nearest City</label>
```

Comment:
The `for` attribute of the label needs to match the `id` of the input it's for, not the `name`. You should update the `id` of the input below this to be `nearestCity`.